Ginger/Client/KoalaClient.php

```diff
@@ -33,21 +114,21 @@ protected function apiCall($endpoint, $params = array(), $method = "GET") {
       $url = $this->url . $endpoint;
       $settings[CURLOPT_POSTFIELDS] = $params;
     }
-    
+
     // Initialisation de cURL
     $ch = curl_init($url);
     curl_setopt_array($ch, $settings);
 
     // Éxécution de la requête
     $result = curl_exec($ch);
-    
+
     // Si erreur d'appel de cron
     if (curl_errno($ch) != 0) {
-      throw new ApiException(503);
+      throw new ApiException(503, self::$error_codes[curl_errno($ch)]);
     }
     // Si erreur http, on la renvoie
     else if (curl_getinfo($ch, CURLINFO_HTTP_CODE) != 200) {
-      throw new ApiException(curl_getinfo($ch, CURLINFO_HTTP_CODE));
+      throw new ApiException(curl_getinfo($ch, CURLINFO_HTTP_CODE), null);
```

Comment:
Indeed, I just thought it would be easier for you to see the changes, but I can remove it :)